pkg/mark/stdlib/stdlib.go

```diff
@@ -374,6 +374,24 @@ func templates(api *confluence.API) (*template.Template, error) {
 			`</ac:structured-macro>`,
 		),
 
+		/* https://confluence.atlassian.com/conf59/page-tree-macro-792499177.html */
+
+		`ac:tree`: text(
+			`<ac:structured-macro ac:name="pagetree" ac:schema-version="1">`,
+			`<ac:parameter ac:name="root">`,
+			`<ac:link>`,
+			`<ri:page ri:content-title="@self"{{or .Title "" }}/>`,
+			`</ac:link>`,
+			`</ac:parameter>`,
+			`<ac:parameter ac:name="sort">{{ or .Sort "" }}</ac:parameter>`,
```

Comment:
Are the defaults all empty here or what does confluence set?

Reply:
These are all empty by default, but can be added if you want to do fancy things.
I reverse engineered what I needed from what I saw in the storage formatted xml. This:
`<p><ac:structured-macro ac:name="pagetree" ac:schema-version="1"><ac:parameter ac:name="root"><ac:link><ri:page ri:content-title="@self" /></ac:link></ac:parameter></ac:structured-macro></p>`

Reply:
if you add all the default values from the macro documentation that you have suggested below, the storage formatted xml you get is quite a bit more verbose, unnecessarily... I actually started with all the suggestions you made, and ended up removing them, because when I insert the macro in confluence, none of those values are present.  what I pr'd was submitted and tested (at least on my version of confluence) to show the same fields that were present when I manually add a page tree macro and set the root to @self.  

I tested against on prem Confluence 7.13.7. 
I can go spin up a test instance in confluence cloud to validate there as well.

Reply:
It actually doesn't seem to work on atlassian cloud so I may have to add them in. Testing further. 
I submitted a readme.md update in my original PR wth the same link ;)

Reply:
I had already cleaned up my test files and I had made a small error.
The code works, as submitted, for all versions of Confluence I can test on prem and in cloud.
No defaults are needed, but I think it is nice to leave the option of specifying those values to others who may have a different use case.